Content.Client/Stylesheets/Colorspace/OkLabColor.cs

```diff
@@ -0,0 +1,106 @@
+﻿using JetBrains.Annotations;
+using Robust.Shared.Utility;
+using Vector4 = Robust.Shared.Maths.Vector4;
+
+namespace Content.Client.Stylesheets.Colorspace;
+
+/// <summary>
+///     Oklab is an alternate color space that more accurately imitates how color actually behaves.
+///     Useful if you want to adjust the lightness/saturation of a color without potentially altering hue.
+///     https://bottosson.github.io/posts/oklab/
+///     Oklch hue and chroma are also provided in <see cref="ColorExtensions"/>
+/// </summary>
+[PublicAPI]
+public struct OklabColor
```

Comment:
we have this in robust toolbox now as of https://github.com/space-wizards/RobustToolbox/pull/6067